pkg/providers/imagefamily/bootstrap/cse_cmd.sh.gtpl

```diff
@@ -116,6 +116,7 @@ HTTPS_PROXY_URLS="{{.HTTPSProxyURLs}}"
 NO_PROXY_URLS="{{.NoProxyURLs}}"
 ENABLE_TLS_BOOTSTRAPPING="{{.TLSBootstrappingEnabled}}"
 ENABLE_SECURE_TLS_BOOTSTRAPPING="{{.SecureTLSBootstrappingEnabled}}"
+ENABLE_KUBELET_SERVING_CERTIFICATE_ROTATION="true"
```

Comment:
not currently safe to set to true in public cloud regions

Reply:
once it is safe to do so in any public region, this should only be set to true iff:
- MC is on k8s 1.27+
- MC has k8s RBAC enabled